Dockerfile

```diff
@@ -3,4 +3,5 @@ FROM rancher/dind:v0.2.0
 COPY ./scripts/bootstrap /scripts/bootstrap
 RUN /scripts/bootstrap
 WORKDIR /source
+VOLUME /target
```

Comment:
The volume is per container. The name of the file is not generated uniquely, so there would be a risk of clobbering if you bind mounted a common directory for multiple test containers concurrently. For what we are doing, I would say its ok. We run validation-tests serially. I don't think a majority of the time right now we will be using the file outside of this environment.